web/src/features/fireWeather/components/maps/WeatherMap.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState, useCallback, useEffect } from 'react'
+import { fromLonLat, get } from 'ol/proj'
+import * as olSource from 'ol/source'
+import GeoJSON from 'ol/format/GeoJSON'
+import { Circle as CircleStyle, Fill, Stroke, Style } from 'ol/style'
+import { FeatureLike } from 'ol/Feature'
+
+import Map from 'features/map/Map'
+import TileLayer from 'features/map/TileLayer'
+import VectorLayer from 'features/map/VectorLayer'
+
+const styles = {
+  Point: new Style({
+    image: new CircleStyle({
+      radius: 4,
+      fill: new Fill({
+        color: '#E59982'
+      }),
+      stroke: new Stroke({
+        color: 'black'
+      })
+    })
+  })
+}
+
+const BC_WEATHER_STATIONS_WEB_FEATURE_SERVICE =
```

Comment:
We could consider making these configurable - that way we can easily switch out layers.